openshift/console
# Bug 1883306: Include repo branch in URL for che icon

**Fixes**: 
https://issues.redhat.com/browse/ODC-4385
#6077 
https://bugzilla.redhat.com/show_bug.cgi?id=1883306



**Analysis / Root cause**: 
The URL generated for the Che editor icon in topology does not consider the repository branch.


**Solution Description**: 
Include repository branch when creating the URL, in the format `${gitURL}/tree/${gitBranch}`.


**Test setup:**
- Install Eclipse Che operator and create a `CheCluster` instance.
- Create an application using Git import flow, optionally adding a different branch in advanced git options.
- Notice the generated URL for che icon in the application's topology node contains the repository branch.


**Browser conformance**:
